packages/common/src/store/pages/chat/slice.ts

```diff
@@ -25,13 +25,15 @@ type UserChatWithMessagesStatus = UserChat & {
   messagesSummary?: TypedCommsResponse<ChatMessage>['summary']
 }
 
+type ChatID = string
+
 type ChatState = {
   chats: EntityState<UserChatWithMessagesStatus> & {
     status: Status
     summary?: TypedCommsResponse<UserChat>['summary']
   }
   messages: Record<
-    string,
+    ChatID,
```

Comment:
I went through and did this once but then realized vscode shows string anyway. Not sure if it adds any real value?

Reply:
I think it's helpful when looking at the slice type here to see that the record key is a ChatId vs say a message UUID or something

Reply:
Yeah I thought so too, but when you hover over the type it shows `string` not `ChatID` in intellisense anyway.

Reply:
Yeah I know, still useful to know in this file IMO. Otherwise could use `Brand`